<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>S3dmApplySymbolDlg</class>
 <widget class="QWidget" name="S3dmApplySymbolDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>符号选择器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>按名称筛选：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="comboLabel"/>
     </item>
     <item>
      <widget class="QListView" name="listView">
       <property name="toolTip">
        <string/>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="labelPreview">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>241</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>编辑</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="2">
           <widget class="QComboBox" name="comboSizeUnit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSize">
            <property name="text">
             <string>大小：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinInterval">
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelRotate">
            <property name="text">
             <string>旋转(°)：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="comboIntervalUnit"/>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinSize">
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QDoubleSpinBox" name="spinRotate">
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelInterval">
            <property name="text">
             <string>间隔：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelOpacity">
            <property name="text">
             <string>不透明度：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QSlider" name="horizontalSlider">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelColor">
            <property name="text">
             <string>背景色：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="buttonColorSelect">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QCheckBox" name="checkBoxColor">
            <property name="text">
             <string>取消背景色</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonYes">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
